src/Listeners/AddUserBioAttribute.php

```diff
@@ -39,9 +45,21 @@ public function __invoke(UserSerializer $serializer, User $user, array $attribut
     {
         $actor = $serializer->getActor();
 
+        $bio = $user->bio ?? '';
+        $isXML = str_starts_with($bio, '<');
```

Comment:
`str_starts_with` is PHP 8.0+.

Might be safer to use Laravel's `Str::startsWith` for the time being.